development-docs/repl-api.md

```diff
@@ -0,0 +1,27 @@
+## How to create your own ts-node powered REPL
+
+- Create ts-node REPL service which includes EvalState
+- Create ts-node service using EvalState-aware `readFile` and `fileExists` implementations from REPL
+- Pass service to repl service
+- Either:
+  - call REPL method start() to start a REPL
+  - create your own node repl but pass it REPL service's eval() function
+
+
+```
+const tsNodeReplService = tsNode.createReplService()
+const {readFile, fileExists} = repl.getStateAwareHostFunctions()
```

Comment:
This looks off - should it be `tsNodeReplService.getStateAwareHostFunctions()`?

Reply:
Looks like `getStateAwareHostFunctions` was left over from the previous commit which renamed `getStateAwareHostFunctions` to `createEvalAwarePartialHost`. So maybe this was the intent?

```suggestion
const {readFile, fileExists} = tsNode.createEvalAwarePartialHost()
```